ci-operator/step-registry/cucushift/upgrade/toimage/cucushift-upgrade-toimage-commands.sh

```diff
@@ -0,0 +1,201 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# Extract oc binary which is supposed to be identical with target release
+# parameters: 1-release pull spec
+function extract_oc(){
```

Comment:
cli can be injected: https://docs.ci.openshift.org/docs/architecture/step-registry/#injecting-the-oc-cli

Reply:
Thanks for pointing it out. I did see this feature, I didn't use this way because, in this case, the oc client is always supposed to be same with the upgrade target release. For instance, a upgrade chain, 4.8->4.9->4.10, when 4.8->4.9 upgrade, the 4.9 oc is required, while 4.9->4.10 upgrade, the 4.10 oc is required. Could the `Injecting the oc CLI` implement this requirement?

Reply:
Yes, the example there does this.